devsetup/README.md

```diff
@@ -92,31 +92,38 @@ After completing the devsetup, attach the crc VM to the default network:
 make crc_attach_default_interface
 ```
 
-Deploy additional VM's for compute nodes:
+Deploy a compute node VM:
 ```
 # Creates edpm-compute-0:
 make edpm_compute
-# Set $EDPM_COMPUTE_SUFFIX to create additional VM's beyond 0:
-make edpm_compute EDPM_COMPUTE_SUFFIX=1
 ```
 
-Edit edpm/edpm-play.yaml and set the compute node VM IP addresses in the
-inventory. The IP address in the inventory (192.168.122.139) needs to be
-changed to the right IP for edpm-compute-0 in the environment. The
-edpm-compute-0 IP can be discovered with the following command:
+Discover IP of the compute node VM:
 ```
 sudo virsh net-dhcp-leases default
 ```
 
-Execute the ansible to configure the compute nodes:
+Execute the ansible to configure the compute node:
 ```
-make edpm_play
+make edpm_play EDPM_COMPUTE_IP=<IP>
 ```
 
+You can also deploy additional compute node VMs:
+```
+# Set $EDPM_COMPUTE_SUFFIX to create additional VM's beyond 0:
+make edpm_compute EDPM_COMPUTE_SUFFIX=1
```

Comment:
we should also use `EDPM_COMPUTE_IP` variable to deploy multiple vms.

Reply:
EDPM_COMPUTE_IP is not used by edpm_compute.
Also, it does not work because users should update a few more items in inventory (such as ips). That's why I suggested modifying the playbook directly as automating generation of inventory is quite complicated.